app/src/main/java/com/sdsmdg/harjot/MusicDNA/Activities/HomeActivity.java

```diff
@@ -78,13 +79,18 @@
 import com.sdsmdg.harjot.MusicDNA.CustomBottomSheetDialogs.CustomLocalBottomSheetDialog;
 import com.sdsmdg.harjot.MusicDNA.CustomViews.CustomLinearGradient;
 import com.sdsmdg.harjot.MusicDNA.Fragments.AboutFragment.AboutFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.AllFoldersFragment.FolderFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.AllPlaylistsFragment.AllPlaylistsFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.EditSongFragment.EditLocalSongFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.EqualizerFragment.EqualizerFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.FavouritesFragment.FavouritesFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.FolderContentFragment.FolderContentFragment;
-import com.sdsmdg.harjot.MusicDNA.Fragments.AllFoldersFragment.FolderFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.AlbumFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.ArtistFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.LocalMusicFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.LocalMusicViewPagerFragment;

```

Comment:
Usually one should avoid unrelated changes in a PR, for a third person referring the purpose of this PR, such changes are noisy. Please undo these import reorders in this PR and include only the additions which are required for haptic feedback.

This shall also help you to see amending and force pushing commits if you haven't tried before.